index.html

```diff
@@ -1742,6 +1779,27 @@ <h2>
                       given |notification|.
                     </p>
                   </li>
+                  <li data-cite="appmanifest">
+                    <p>
+                      If |appBadgeSet| is false and the web application associated with the <a>push
+                      subscription</a> is an [=installed web application=]:
```

Comment:
We should use |subscription| here, I think? Maybe

> If appBadgeSet is false and |subscription| has an associated installed web application:

Reply:
Though it would be nice to also support this outside of installed web applications. But I suppose that's a separate change.

Reply:
Setting a badge is pretty tightly bound to installation at the moment, as that's how it's been implemented by everyone. We could change this if this changes.